packages/integration-karma/test/light-dom/multiple-templates/x/multi/multi.js

```diff
@@ -3,7 +3,7 @@ import A from './a.html';
 import B from './b.html';
 
 export default class Multi extends LightningElement {
-    static shadow = false;
+    static renderMode = 'light';
```

Comment:
Looks like this change didn't result in any test changes even though it's changing from synthetic mode to light dom mode. Is it being used in any tests?

Reply:
`static shadow = false` is also Light DOM, but older way of indicating light DOM. We changed it here (https://github.com/salesforce/lwc/pull/2344) to `static renderMode = 'light'`.

Reply:
Yes, but we already deprecated that property, right? Doesn't that mean that if you currently have a component with `static shadow = false`, it runs in synthetic mode?

Reply:
I'm pretty sure `static shadow = false` does nothing nowadays.

Reply:
I just took a look at the test and it looks like it should fail unless the component is a Light DOM component. Can someone help explain how it was previously passing as a component running in synthetic mode?

Reply:
I rebased with master. This diff isn't there. It was failing earlier, I fixed it a few days ago. https://github.com/salesforce/lwc/commit/8cd6aa6f16a68db6892344b4bf4db15c0c23544a